packages/vite/src/node/plugins/worker.ts

```diff
@@ -57,7 +56,7 @@ export function webWorkerPlugin(config: ResolvedConfig): Plugin {
         const rollup = require('rollup') as typeof Rollup
         const bundle = await rollup.rollup({
           input: cleanUrl(id),
-          plugins: await resolvePlugins({ ...config }, [], [], []),
+          plugins: config.plugins.map((plugin) => plugin),
```

Comment:
@patak-dev mentioned this line in another PR and that it is dangerous to change it because there is somehow a design-decision in this code :thinking:

Reply:
Yes, @poyoho, please check the discussion in https://github.com/vitejs/vite/pull/5390. I'll comment on the main thread here.